documentation/modules/ROOT/pages/install.adoc

```diff
@@ -70,10 +70,10 @@ endif::[]
 == Using a Debezium Connector
 
 To use a connector to produce change events for a particular source server/cluster, simply create a configuration file for the
-xref:connectors/mysql.adoc#deploying-a-connector[MySQL Connector],
+xref:assemblies/cdc-mysql-connector/as_deploy-the-mysql-connector.adoc[MySQL Connector],
 xref:connectors/postgresql.adoc#deploying-a-connector[Postgres Connector],
-xref:connectors/mongodb.adoc#deploying-a-connector[MongoDB Connector],
-xref:connectors/sqlserver.adoc#deploying-a-connector[SQL Server Connector],
+xref:connectors/mongodb.adoc#mongodb-deploying-a-connector[MongoDB Connector],
+xref:connectors/sqlserver.adoc#sqlserver-deploying-a-connector[SQL Server Connector],
 xref:connectors/oracle.adoc#deploying-a-connector[Oracle Connector],
```

Comment:
@ahus1, already merged this, but one question: why did you add the prefix ("mongodb", "sqlserver") to the two above, but not these two?

Reply:
@gunnarmorling - I assume "these two" apply to oracle.adoc and db2.adoc. 

The documentation for the different connectors is inconsistent in its structure: some named the anchor with the prefix (mongodb, sqlserver), some didn't (oracle, db2, cassandra). 

What I did here: fix the broken anchors by pointing to the most appropriate existing anchor ID. What I didn't do: restructure/rename anchor IDs.

From an AsciiDoc perspective IDs need to be unique within a document. For Antora each page is a document. If you combine the documentation in a different way (maybe the downstream product?) to a big document, then it might be handy to have unique IDs across all the AsciiDoc files.

Reply:
>  I assume "these two" apply to oracle.adoc and db2.adoc.

Yes, exactly.

> What I did here: fix the broken anchors by pointing to the most appropriate existing anchor ID. What I didn't do: restructure/rename anchor IDs.

Ok, gotcha. Thanks for clarifying. So uniqueness within each connector documentation should be enough, we don't create one large document with all the documentation at once. Indeed I think the slight inconsistency originates from downstream work, as the docs team has added the prefix to those files which feed into the downstream product.

I.e. all good, thanks a lot again!